Swinject/Container.swift

```diff
@@ -194,7 +193,7 @@ extension Container: Resolvable {
         }
         
         let resolvedInstance = invoker(entry.factory as! Factory)
-        if usesPool {
+        if (usesPool) {
```

Comment:
Control Statement Parentheses Violation: if statements shouldn't wrap their conditionals in parentheses.